packages/office-ui-fabric-react/src/components/Separator/examples/Separator.Basic.Example.tsx

```diff
@@ -0,0 +1,66 @@
+import * as React from 'react';
+import { Separator } from 'office-ui-fabric-react/lib/Separator';
+import { mergeStyles } from 'office-ui-fabric-react/lib/Styling';
+import { Stack } from 'office-ui-fabric-react/lib/Stack';
+import { Text } from 'office-ui-fabric-react/lib/Text';
+
+export class SeparatorBasicExample extends React.Component<{}, {}> {
+  public render(): JSX.Element {
+    const message = 'Today';
+
+    const verticalStyle = mergeStyles({
+      height: '200px'
+    });
+
+    return (
```

Comment:
![Screen Shot 2019-03-28 at 2 54 26 PM](https://user-images.githubusercontent.com/29514833/55195334-7356e000-5169-11e9-8aad-e6deea17fe5f.png)
This example page is more about stack than about separator 😄  just an observation...

Reply:
Valid point! Maybe it would be worth splitting these up into a couple sections. I didn't want to overwhelm the page with too many separate example files, but that would definitely make finding the actual Separator code easier

Reply:
Actually it seems these nested stacks only have one or two configurations, I think it'd be better to reduce the duplication by creating a component wrapper helper with the Stacks that renders the Separator children passed in. That would also help de-emphasize the usage of Stack.

Reply:
There's an example [here](https://github.com/OfficeDev/office-ui-fabric-react/blob/30bea805e3c1fc69010e827799e068a8813fa689/packages/experiments/src/components/Button/examples/Button.Example.tsx#L13). I think it could be adapted to two-level `Stacks` too.

Reply:
Thanks Jason!